### PR TITLE
Add more documentation for the macOS installation

### PR DESCRIPTION
**Pull request by Emin017**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,13 @@ dnf install meson gperf flex glib2-devel gcc gcc-c++ gtk3-devel \
             gobject-introspection-devel desktop-file-utils tcl
 ```
 
+MacOS:
+```sh
```
Comment (rfuest) [inline README.md:24]:
```suggestion
macOS:

```sh
```

File: docs/install/mac.md [hunk 2]
```diff
@@ -9,4 +9,69 @@ However, this version is not compatible with current versions of macOS.
...
+```shell
+# Use --prefix to specify the installation path
+meson setup build --prefix=/opt
+# Or install to default path (/usr/local):
+# meson setup build
+meson compile -C build # Start compile
+sudo meson install -C build # Install gtkwave
+```
+
+gtkwave is now installed on your macOS. You can start it in your terminal, run:
```
Comment (rfuest) [inline docs/install/mac.md:69]:
```suggestion
GTKWave is now installed on your macOS. You can start it in your terminal, run:
```

File: docs/install/mac.md [hunk 3]
```diff
@@ -9,4 +9,69 @@ However, this version is not compatible with current versions of macOS.
...
+
+```shell
+gtkwave
+```
+
+## Troubleshooting
+
+If you encounter any errors during the installation process, please refer to the GitHub issue for additional help and troubleshooting tips.
```
Comment (rfuest) [inline docs/install/mac.md:77]: I think we should repeat the link to the GH issue here:
```suggestion
If you encounter any errors during the installation process, please refer to the [GitHub issue](https://github.com/gtkwave/gtkwave/issues/250) for additional help and troubleshooting tips.
```

**Review by rfuest — CHANGES_REQUESTED** [review]
Thanks, this looks great. I only have some minor nits.

**Comment by Emin017** [comment]
I pushed the fixed commits.
Please Let me know if there's anything else that needs to be changed :)

**Review by rfuest — APPROVED** [review]
LGTM, thanks again.

**Pull request merged**